string                 name
float64[]              Tstart

---

geometry_msgs/Pose     pose
bool                   found
